modules/browser/BrowserCapabilities.ts

```diff
@@ -27,32 +38,146 @@ export default class BrowserCapabilities extends BrowserDetection {
      *
      * @return {boolean} <tt>true</tt> if the current browser supports this strategy or <tt>false</tt> otherwise.
      */
-    doesVideoMuteByStreamRemove() {
+    doesVideoMuteByStreamRemove(): boolean {
         return this.isChromiumBased() || this.isWebKitBased() || this.isFirefox();
     }
 
+    // Define the isChromiumBased method
```

Comment:
All os these are defined in the base class, which is defined in the `@jitsi/js-utils` package.